recipes-core/dist-nilrt/files/nilrt-gateway-install

```diff
@@ -37,6 +43,19 @@ function die() {
     exit 1
 }
 
+# Download dist-nilrt-efi-ab.ipk into /payload of nirootfs partition
+NIROOTFS_MOUNT=$(mktemp -d "/var/volatile/niroot-XXXXXXX")
+mount -L nirootfs $NIROOTFS_MOUNT || die "Unable to mount nirootfs."
+
+rm -rf $NIROOTFS_MOUNT/payload
+mkdir $NIROOTFS_MOUNT/payload || die "Unable to create payload directory."
+
+pushd $NIROOTFS_MOUNT/payload
+opkg update > /dev/null
+opkg download dist-nilrt-efi-ab > /dev/null || (popd; die "Unable to download dist-nilrt-efi-ab. Check if disk has sufficient space.")
+ls dist-nilrt-efi-ab*.ipk &> /dev/null || (popd; die "Unable to download dist-nilrt-efi-ab. Check if feeds contain dist-nilrt-efi-ab package.")
```

Comment:
> Is there a reason to use ls here instead of test -e?
Wasn't aware of test -e. Will change to it.

> Also the fact that opkg returns 0 when it can't satisfy a download request is a bug. I'll fix it upstream when I get some time. But this is fine until then.
Thanks